cmd/vic-init/ops.go

```diff
@@ -83,6 +85,20 @@ func (t *operations) SetHostname(name string, aliases ...string) error {
 	return t.BaseOperations.SetHostname(aliases[0])
 }
 
+func (t *operations) SetupFirewall() error {
+	// get the public interface name
+	l, err := netlink.LinkByName("public")
```

Comment:
Can we have constants for these "public" and others? I see it is getting used too frequent, thus it becomes error prone.